### DISCUSSION TITLE
Simple landing

### DISCUSSION THREAD

**Discussion by Dhathrikrishna** [community/community#135241] [Mobile]
### Select Topic Area

Show & Tell

### Body

<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="A brief description of your product or service">
    <title>Landing Page</title>
    <link rel="stylesheet" href="styles.css">
</head>
<body>

    <header class="header">
        <h1>Your Product or Service Headline</h1>
        <p class="description">A brief description of your product or service, highlighting key features or benefits.</p>
        <a href="#contact" class="cta-button">Get Started</a>
    </header>

    <section class="visual">
        <img src="your-image.jpg" alt="Engaging visual related to your product or service">
    </section>

    <section id="contact" class="contact-form">
        <h2>Contact Us</h2>
        <form action="submit_form.php" method="post">
            <label for="name">Name:</label>
            <input type="text" id="name" name="name" required>
            
            <label for="email">Email:</label>
            <input type="email" id="email" name="email" required>
            
            <label for="message">Message:</label>
            <textarea id="message" name="message" required></textarea>
            
            <button type="submit">Submit</button>
        </form>
    </section>

</body>
</html>

**Comment by everforyou** [comment]
Hello @Dhathrikrishna 

To create a simple landing page based on the provided HTML structure, here’s a corrected and complete version of your code:

```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Landing Page</title>
    <link rel="stylesheet" href="styles.css"> <!-- Optional: Link to CSS for styles -->
</head>
<body>
    <header class="header">
        <h1>Your Product or Service Headline</h1>
        <p class="description">A brief description of your product or service, highlighting key features or benefits.</p>
        <a href="#contact" class="cta-button">Get Started</a>
    </header>

    <section class="visual">
        <img src="your-image.jpg" alt="Engaging visual related to your product or service">
    </section>

    <section id="contact" class="contact-form">
        <h2>Contact Us</h2>
        <form action="submit_form.php" method="post">
            <label for="name">Name:</label>
            <input type="text" id="name" name="name" required>

            <label for="email">Email:</label>
            <input type="email" id="email" name="email" required>

            <label for="message">Message:</label>
            <textarea id="message" name="message" required></textarea>

            <button type="submit">Submit</button>
        </form>
    </section>
</body>
</html>
```

### Key Components:
1. **Header**: Contains the main headline and description.
2. **Visual Section**: Displays an image related to the product or service.
3. **Contact Form**: Collects user information.

### Notes:
- Make sure to replace `"your-image.jpg"` with the actual path to your image.
- You can link a CSS file for styling (optional).
- Ensure the action in the form points to a valid server-side script to handle submissions (`submit_form.php` in this case).

**Reply by Pioneer220** [reply]
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>GitHub Achievements Showcase</title>
    <style>
        .badge-container {
            display: flex;
            justify-content: center;
            flex-wrap: wrap;
            gap: 20px;
            padding: 20px;
        }
        .badge {
            width: 296px;
            transition: transform 0.3s ease;
        }
        .badge:hover {
            transform: scale(1.05);
        }
        .badge-section {
            text-align: center;
            margin: 20px 0;
        }
        .achievement-description {
            max-width: 700px;
            margin: 0 auto;
            padding: 20px;
        }
    </style>
</head>
<body>
    <header class="header">
        <h1>GitHub Achievement Collection</h1>
        <p class="description">Showcase your GitHub achievements with style</p>
    </header>

    <div class="badge-container">
        <div class="badge-section">
            <a href="#pull-shark">
                <img class="badge" src="assets/badges/PullShark.png" alt="Pull Shark Achievement">
            </a>
            <div class="achievement-description">
                <h2>Pull Shark</h2>
                <p>Earned by completing 2 merged pull requests</p>
            </div>
        </div>

        <div class="badge-section">
            <a href="#starstruck">
                <img class="badge" src="assets/badges/Starstruck.png" alt="Starstruck Achievement">
            </a>
            <div class="achievement-description">
                <h2>Starstruck</h2>
                <p>Repository reached 16 stars</p>
            </div>
        </div>

        <div class="badge-section">
            <a href="#yolo">
                <img class="badge" src="assets/badges/Yolo.png" alt="Yolo Achievement">
            </a>
            <div class="achievement-description">
                <h2>YOLO</h2>
                <p>Merged pull request without review</p>
            </div>
        </div>
    </div>

    <section class="contact-form">
        <h2>Share Your Achievements</h2>
        <form action="submit_form.php" method="post">
            <label for="github-username">GitHub Username:</label>
            <input type="text" id="github-username" name="github-username" required>

            <label for="achievement">Achievement Earned:</label>
            <select id="achievement" name="achievement" required>
                <option value="pull-shark">Pull Shark</option>
                <option value="starstruck">Starstruck</option>
                <option value="yolo">YOLO</option>
            </select>

            <button type="submit">Submit Achievement</button>
        </form>
    </section>
</body>
</html>

I would like to suggest this code.
